### PR TITLE
kucoinfutures add average to order info

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: js/kucoinfutures.js [hunk 1]
```diff
@@ -1144,6 +1144,12 @@ module.exports = class kucoinfutures extends kucoin {
         const rawCost = this.safeString2 (order, 'dealFunds', 'filledValue');
         const leverage = this.safeString (order, 'leverage');
         const cost = Precise.stringDiv (rawCost, leverage);
+        let average = undefined;
+        if (market['linear']) {
+            average = Precise.stringDiv (rawCost, Precise.stringMul (market['contractSize'], filled));
```
Comment (kroitor) [inline js/kucoinfutures.js:1149]: What if `filled` is 0?

Comment (kroitor) [reply js/kucoinfutures.js:1149]: Nevermind, should be ok.

Comment (frosty00) [reply js/kucoinfutures.js:1149]: then average will be undefined for linear futueres and 0 for inverse futures

Comment (kroitor) [reply js/kucoinfutures.js:1149]: I think we might need to set it to undefined for inverse futures, not 0.

**Pull request merged**